docs/v3/openapi.json

```diff
@@ -6611,7 +6682,7 @@
                     }
                 ]
             },
-            "put": {
+            "patch": {
```

Comment:
does this mean that `"/templates/{template_id}/systems"` PUT won't be documented but will still work?

Reply:
exactly :)